static/js/components/tiles/sv_ranking_units.tsx

```diff
@@ -21,7 +21,6 @@ import React, { RefObject, useRef } from "react";
 
 import { VisType } from "../../apps/visualization/vis_type_configs";
 import { URL_PATH } from "../../constants/app/visualization_constants";
-import { ASYNC_ELEMENT_CLASS } from "../../constants/css_constants";
```

Comment:
For my understanding, why remove this?  There are other places that use it too.

Reply:
ASYNC_ELEMENT_CLASS in this file is a dup. Wherever it's used, the child html also have this class added and should be preferred.